package NYT.NChunkClient.NProto;

////////////////////////////////////////////////////////////////////////////////

message TDataStatistics
{
    required int64 uncompressed_data_size = 1;
    required int64 compressed_data_size = 2;
    required int64 row_count = 3;
    required int64 chunk_count = 4;
}

////////////////////////////////////////////////////////////////////////////////
